lua/feline/providers/lsp.lua

```diff
@@ -1,59 +1,67 @@
 local M = {}
 
-function M.is_lsp_attached()
-    return next(vim.lsp.buf_get_clients()) ~= nil
+local lsp = vim.lsp
```

Comment:
https://github.com/famiu/feline.nvim/pull/33 was merged to stop loading it eagerly.

Reply:
Oops, forgot about that. Fixed now